doc/htmldoc/conf.py

```diff
@@ -253,6 +256,7 @@ def setup(app):
     app.add_css_file('css/custom.css')
     app.add_css_file('css/pygments.css')
     app.add_js_file("js/custom.js")
+    app.add_js_file("js/mermaid.js")
```

Comment:
Is this really necessary? Isn't the JavaScript automatically integrated when the plug-in is activated?